pages/docs/vi/vi-headless.md

```diff
@@ -0,0 +1,91 @@
+# First Steps
+
+## Headless Set up
+&nbsp;
+Don’t have an extra keyboard or HDMI cable? Here’s how to do a headless [Treehouses](http://download.treehouses.io/) install on your Pi.
+&nbsp;
+Here is the list of hardware requirements once again for you to refer to.
+&nbsp;
+#### Hardware requirement review:
+1.  A Raspberry Pi 4
+1.  A power cord for the Pi
+	- Standard micro USB power cord: 2.4A for Pi 3 and Pi 3b+, 1.2A for Pi Zero W, or 15.3W for Pi 4
```

Comment:
Pi 4 is 3A, Pi 3 B/B+ is 2.5 according to https://www.raspberrypi.org/documentation/faqs/#pi-power